tests/dialects/test_cf.py

```diff
@@ -12,11 +12,12 @@ def test_branch():
 
     block0 = Block([a, b, c])
     br0 = Branch.get(block0)
+    ops = list(br0.successors[0].ops)
 
     assert br0.successors[0] is block0
-    assert br0.successors[0].ops[0] is a
-    assert br0.successors[0].ops[1] is b
-    assert br0.successors[0].ops[2] is c
+    assert ops[0] is a
```

Comment:
I opted against adding an `__index__` to the `_BlockOps`, because code like this would be accidentally O(n²)